### PR TITLE
Add isAbstract to AbstractClassAssert

### LINKED ISSUE #1416: ClassAssert.isAbstract
accidentally forgot the abstract modifier on my abstract base class, oops, went to write a regression test, and no `isAbstract` on `ClassAssert`, this is not critical to me, but might be useful.

### PR DESCRIPTION
**Pull request by epeee**
#### Check List:
* Fixes #1416 
* Unit tests : YES
* Javadoc with a code example (API only) : YES

### CODE REVIEW

File: src/test/java/org/assertj/core/internal/classes/Classes_assertIsAbstract.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import static org.assertj.core.error.ShouldBeAbstract.shouldBeAbstract;
+import static org.assertj.core.test.TestData.someInfo;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+
+class Classes_assertIsAbstract extends ClassesBaseTest {
+
+  @Test
+  void should_fail_if_actual_is_null() {
+    actual = null;
+    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> classes.assertIsAbstract(someInfo(), actual))
```
Comment (joel-costigliola) [inline src/test/java/org/assertj/core/internal/classes/Classes_assertIsAbstract.java]: use https://github.com/joel-costigliola/assertj-core/blob/master/src/test/java/org/assertj/core/util/AssertionsUtil.java#L30

File: src/test/java/org/assertj/core/internal/classes/Classes_assertIsAbstract.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  @Test
+  void should_pass_if_actual_is_abstract() {
+    actual = AbstractClassAssert.class;
+    classes.assertIsAbstract(someInfo(), actual);
+  }
+
+  @Test()
+  void should_fail_if_actual_is_not_abstract() {
+    actual = Classes_assertIsAbstract.class;
+    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> classes.assertIsAbstract(someInfo(), actual))
```
Comment (joel-costigliola) [inline src/test/java/org/assertj/core/internal/classes/Classes_assertIsAbstract.java]: same here: https://github.com/joel-costigliola/assertj-core/blob/master/src/test/java/org/assertj/core/util/AssertionsUtil.java#L30

**Comment by joel-costigliola** [comment]
@epeee two minor comments, thanks for the good work!

File: src/test/java/org/assertj/core/internal/classes/Classes_assertIsAbstract.java (hunk 1 above)
Comment (epeee) [reply src/test/java/org/assertj/core/internal/classes/Classes_assertIsAbstract.java]: thank you, I completely forgot about this one.

**Comment by joel-costigliola** [comment]
Integrated thanks @epeee !

**Pull request closed without merging**